### DISCUSSION TITLE
Mis practicas

### DISCUSSION THREAD

**Discussion by Paupiarce** [Launch-X-Latam/MisionFrontEnd#415] [Show and tell]
Holis, acá les comparto mis practicas de Front-end. No entiendo realmente en dónde hay que subirlas pero bueno, hice los repositorios
https://github.com/Paupiarce/practica-css
https://github.com/Paupiarce/practica-JS
https://github.com/Paupiarce/Practica-HTML

**Comment by OscarJG20** [comment]
Hola!
Creo que has dado unos buenos inicios, sobre todo con la página de vacunas.
Si quieres que lo demás visualicen tus avances podrías usar el servicio gratuito de Netlify.

Como una actividad algo más personal yo creo que podrías elegir uno de tus proyectos y ese pulirlo para que sea bien presentado en móvil y escritorio, yo elegiría el de las vacunas.

Chaito :D

**Reply by Paupiarce** [reply]
Muchísimas gracias Oscar! Eso me motiva a seguir para adelante :'D
